AFAICS, the added tests don't pass:
```
[ERROR]   SentryLoggerEnvironmentOptionTests.sentryLoggerEnvironmentOptionTest:27 
Expecting:
 <"">
to be equal to:
 <"test-environment">
but was not.
[ERROR]   SentryLoggerReleaseOptionTests.sentryLoggerEnvironmentOptionTest:27 
Expecting:
 <"">
to be equal to:
 <"releaseABC">
but was not.
```

--- a/extensions/logging-sentry/deployment/src/test/resources/application-sentry-logger-environment-option.properties
+++ b/extensions/logging-sentry/deployment/src/test/resources/application-sentry-logger-environment-option.properties
@@ -2,4 +2,4 @@ quarkus.log.sentry=true
 quarkus.log.sentry.dsn=https://123@test.io/22222
 quarkus.log.sentry.level=TRACE
 quarkus.log.sentry.in-app-packages=io.quarkus.logging.sentry,org.test
-quarkus.log.sentry.release=releaseABC
+quarkus.log.sentry.environment=test-environment

--- a/extensions/logging-sentry/deployment/src/test/resources/application-sentry-logger-release-option.properties
+++ b/extensions/logging-sentry/deployment/src/test/resources/application-sentry-logger-release-option.properties
@@ -2,4 +2,4 @@ quarkus.log.sentry=true
 quarkus.log.sentry.dsn=https://123@test.io/22222
 quarkus.log.sentry.level=TRACE
 quarkus.log.sentry.in-app-packages=io.quarkus.logging.sentry,org.test
-quarkus.log.sentry.environment=test-environment
+quarkus.log.sentry.release=releaseABC

--- a/extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java
+++ b/extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfig.java
@@ -51,7 +51,7 @@ public class SentryConfig {
     /**
      * Environment
      *
-     * As of Sentry 9, you can easily filter issues, releases, and user feedback by environment.
+     * With Sentry you can easily filter issues, releases, and user feedback by environment.
      * The environment filter on sentry affects all issue-related metrics like count of users affected, times series graphs,
      * and event count.
      * By setting the environment option, an environment tag will be added to each new issue sent to Sentry.
@@ -59,8 +59,6 @@ public class SentryConfig {
      * There are a few restrictions:
      * -> the environment name cannot contain newlines or spaces, cannot be the string “None” or exceed 64 characters.
      *
-     * For further detail, please refer to the sentry environment option documentation:
-     * https://docs.sentry.io/enriching-error-data/environments/
      */
     @ConfigItem
     public Optional<String> environment;
@@ -75,8 +73,6 @@ public class SentryConfig {
      * - Resolve issues by including the issue number in your commit message
      * - Receive email notifications when your code gets deployed
      *
-     * For further detail, please refer to the sentry release option documentation:
-     * https://docs.sentry.io/workflow/releases/
      */
     @ConfigItem
     public Optional<String> release;

--- a/extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfigProvider.java
+++ b/extensions/logging-sentry/runtime/src/main/java/io/quarkus/logging/sentry/SentryConfigProvider.java
@@ -26,9 +26,9 @@ class SentryConfigProvider implements ConfigurationProvider {
                         .filter(s -> !Objects.equals(s, "*"))
                         .orElse("");
             case DefaultSentryClientFactory.ENVIRONMENT_OPTION:
-                return config.environment.orElse("");
+                return config.environment.orElse(null);
             case DefaultSentryClientFactory.RELEASE_OPTION:
-                return config.release.orElse("");
+                return config.release.orElse(null);
             // New SentryConfig options should be mapped here
             default:
                 return null;

